spec/features/organization_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.feature "Organization management", type: :feature do
+  before do
+    sign_in(@user)
+  end
+  let!(:url_prefix) { "/#{@organization.to_param}"}
+  scenario "When editing their organization, the user can enter a URL with or without http://" do
+  	visit url_prefix + '/organization/edit'
+    fill_in "Url", with: "www.diaperbase.com"
+    click_button "Update"
+
+    expect(page.find('.url .error')).to have_content "www.example.com"
+
+    fill_in "Url", with: "http://www.diaperbase.com"
+    click_button "Update"
+    expect(page.find('.flash.success')).to have_content "pdated"
```

Comment:
The two assertions don't match up with the `scenario` description. The scenario reads that the user *should* be able to enter a URL with *or* without `http://` but the tests look like they are asserting a url without the `http://` is still invalid.